moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -315,6 +315,7 @@ export const useColumns = ({ hiddenColumns }) => {
       {
         headerName: "Substantial completion date",
         field: "substantial_completion_date",
+        description: "The earliest confirmed start date of a project phase that has a simple phase name of Complete (Complete or Post-Construction)",
```

Comment:
@tillyw Could you update the description to say `Post-construction` so it matches the capitalization in the [data dictionary](https://mobility.austin.gov/moped/dev/lookups#moped-phases)? I made a typo and then copy-pasta'd into every issue involving this date? Just fixed those

Reply:
done!